pkg/client/images.go

```diff
@@ -22,22 +22,22 @@ import (
 // ImagesClient defines the image client interface
 type ImagesClient interface {
 	// Images
-	CreateImagectx(ctx context.Context, image *v1.Image) (*v1.Image, error)
-	DeleteImage(ctx context.Context, imageName string) (*v1.Image, error)
-	UpdateImage(ctx context.Context, image *v1.Image) (*v1.Image, error)
-	GetImage(ctx context.Context, imageName string) (*v1.Image, error)
-	ListImages(ctx context.Context) ([]v1.Image, error)
+	CreateImage(ctx context.Context, organizationID string, image *v1.Image) (*v1.Image, error)
```

Comment:
Does org ID belong in the context rather than an arg?

Reply:
I will make it easy to pass orgID in the clients in a subsequent PR. 

On the other comment, I could add orgID to model but since orgID is passed as a header in the request, we still have to set it explicitly on the model (since the request body won't include the orgID).